src/main/java/com/board/config/boardconfiggui/Main.java

```diff
@@ -0,0 +1,7 @@
+package com.board.config.boardconfiggui;
+
+public class Main {
```

Comment:
Why do I need another Main class, this can be removed

Reply:
The main class should not be extended for the creation of exe 
So created another Main file